tap_zoom/schemas/report_webinars.json

```diff
@@ -116,6 +116,32 @@
         "null",
         "string"
       ]
+    },
```

Comment:
`user_id` field is not mentioned in [docs](https://developers.zoom.us/docs/api/rest/reference/zoom-api/methods/#operation/reportWebinarDetails). are we generating during tap run ?